content/blog/getting-started-on-digitalocean-with-pulumi/index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Getting Started on DigitalOcean with Pulumi"
+authors: ["paul-stack"]
+tags: ["DigitalOcean", "TypeScript"]
+date: "2019-07-18"
+
+meta_image: "RELATIVE_TO_PAGE/Feature.jpg"
+---
+
+Pulumi recently added support for managing [DigitalOcean](https://www.digitalocean.com/) resources. This article will 
+show you how to deploy some load balanced Droplets on DigitalOcean using Pulumi.
+
+To get started, let's create a new Pulumi program written in TypeScript:
+
+```bash
+ $ mkdir infra && cd infra && pulumi new typescript
```

Comment:
We've been showing this as 2 steps. It separates the standard Unix stuff from the part that is Pulumi-specific.
```suggestion
 $ mkdir infra && cd infra
 $ pulumi new typescript
```